src/components/DatePicker/index.android.js

```diff
@@ -73,7 +72,7 @@ class DatePicker extends React.Component {
                 />
                 {this.state.isPickerVisible && (
                     <RNDatePicker
-                        value={this.props.value || this.props.defaultValue ? moment(this.props.value || this.props.defaultValue).toDate() : new Date()}
+                        value={this.props.value || this.props.defaultValue ? new Date(this.props.value || this.props.defaultValue) : new Date()}
```

Comment:
I understand that it was, but could we improve this condition?
```suggestion
                        value={new Date(this.props.value || this.props.defaultValue || ''/undefined)}
```

Reply:
new Date("") or new Date(undefined) will cause Invalid Date, thats why this verification is here

Reply:
Okay 👍